### PR TITLE
Update WhatsApp declarations

### PR DESCRIPTION
**Pull request by martinratinaud**

### CODE REVIEW

**Comment by clementbiron** [comment]
In this case i'm wondering about the relevance of deleting the table of contents of a document:
- the title `Table of Contents` remains displayed in the `Privacy Policy.md` file
- using `._-pk > ul:nth-child(3)` selector in `remove` key is not the most reliable if the content of the document evolves with the addition of `ul` element

---

See below the table of content of the privacy policy of WhatsApp.
image

@MattiSG, @martinratinaud  What do you think ?

**Comment by MattiSG** [comment]
I think it's not a problem if it stays included, and that the least we remove / select the least maintenance we have to do, so I'd be in favour of not removing it.
I also think it's fine if it stays removed 🙂

**Comment by clementbiron** [comment]
I'm agree to keep include this table of content 👍

File: declarations/WhatsApp.json [hunk 1]
```diff
@@ -8,8 +8,13 @@
...
     "Privacy Policy": {
       "fetch": "https://www.whatsapp.com/legal/updates/privacy-policy-eea?lang=en",
-      "select": "._2zry",
-      "remove": "[href='#top-of-page'],._-pk > ul:nth-child(3)"
+      "select": [
+        "#content-wrapper"
+      ],
+      "remove": [
+        "[href='#top-of-page']",
+        "._-pk > ul:nth-child(3)"
```
Comment (clementbiron) [inline declarations/WhatsApp.json:20]: As discuss with Matti, i think you could stop removing the table of content.

Comment (martinratinaud) [reply declarations/WhatsApp.json]: I kept it then

**Review by clementbiron — APPROVED** [review]

**Pull request merged**